cmd/webhook/main.go

```diff
@@ -77,7 +78,9 @@ func main() {
 	kubeConfig := config.GetConfigOrDie()
 
 	// Setup client
-	client, err := kubernetes.NewForConfig(kubeConfig)
+	coreKubeConfig := rest.CopyConfig(kubeConfig)
```

Comment:
I am curious why do we need to make a copy here?

Reply:
Because, later on in `main` function, we pass the `kubeConfig` variable to `manager.New` function call documentation of which specifies that if `kubeConfig` doesn't specify the `ContentType`, it will set it to protobuf for core K8s APIs and to JSON for custom resources: https://github.com/kubernetes-sigs/controller-runtime/blob/2eb879f25c4829825e6f2511cda035de41ec7030/pkg/manager/manager.go#L317-L321.

This is good for us, because:
- CRD doesn't support protobuf encoding (so passing `kubeConfig` with the `ContentType` field set to protobuf would prospectively break it if we start talking to K8s API about CRs),
- it sets protobuf encoding for the core K8s APIs automatically anyway - which was the intention of this PR.

Reply:
Sounds good, thank you for the PR!